zmoney/Models/StateManager.swift

```diff
@@ -78,7 +78,7 @@ struct StateManager: StateHolder, StateApplicable {
     }
 
     private func addViewAndBringToFront(_ view: UIView) {
-        view.frame = loadedView.bounds
+        view.frame = loadedView.frame
```

Comment:
Здесь не очень брать `frame` у дочерней `view`, так как он запросто может поменяться. Лучше здесь завязывать `frame` вьюшек со стейтом на корневую `view`.

```suggestion
        view.frame = rootView.bounds
```